tarsier/core.py

```diff
@@ -47,12 +48,24 @@ def _run_ocr(self, image: bytes) -> str:
         page_text = self._ocr_service.format_text(ocr_text)
         return page_text
 
-    async def _tag_page(self, adapter: BrowserAdapter) -> Dict[int, str]:
+    async def _tag_page(
+        self, adapter: BrowserAdapter, tagUninteractableText: bool = False
+    ) -> Dict[int, str]:
         await adapter.run_js(self._js_utils)
 
-        script = "tagifyWebpage(null, null);"
+        script = f"tagifyWebpage({str(tagUninteractableText).lower()});"
         if isinstance(adapter, SeleniumAdapter):
             script = f"return window.{script}"
 
-        _, tag_to_xpath = await adapter.run_js(script)
+        tag_to_xpath = await adapter.run_js(script)
         return {int(key): value for key, value in tag_to_xpath.items()}
+
+    async def _remove_tags(self, adapter: BrowserAdapter) -> None:
+        # await adapter.run_js(self._js_utils)
```

Comment:
delete?

Reply:
Why? We call this after tagging the page

Reply:
Oh do you mean the comment? Yes I think we could—since it's private and only ever called after running _tag_page, the utils should alr be loaded

Reply:
Yeah just the comment